packages/web3-eth-accounts/src/index.js

```diff
@@ -307,7 +307,7 @@ Accounts.prototype.decrypt = function (v3Keystore, password, nonStrict) {
         kdfparams = json.crypto.kdfparams;
 
         // FIXME: support progress reporting callback
```

Comment:
What about this "FIXME" comment? Is it still valid?

Reply:
I think it can be removed since there's not much point in trying to make use of the progress callback of `scryptsy` and Node's built-in doesn't have such a thing.